plugins/msr/meson.build

```diff
@@ -4,9 +4,11 @@ install_data(['msr.quirk'],
   install_dir: join_paths(datadir, 'fwupd', 'quirks.d')
 )
 
+if get_option('systemd')
 install_data(['fwupd-msr.conf'],
-  install_dir: join_paths(sysconfdir, 'modules-load.d')
+  install_dir: systemd_modules_load_dir,
 )
+endif
```

Comment:
Hmm, so what do we do on a build _without_ systemd?  It just doesn't get to load the configuration file?  Should we have a fallback hardcoded path?

Reply:
> so what do we do on a build without systemd

I'm not sure I care; the module load stuff is implemented by systemd-udev these days anyway....

Reply:
What about maybe emitting a warning in that else condition in this case so it's not a huge surprise for anyone?

Reply:
I guess that makes sense; the alternative is that we make the module-load-using plugins also depend on systemd...

Reply:
I think making them depend on systemd is probably a bit much.  Those modules could be compiled into a kernel, or they could have been previously loaded by something else.